test/tests/p5.Envelope.js

```diff
@@ -0,0 +1,56 @@
+let expect = chai.expect;
+
+describe('p5.Envelop', function () {
+  let sf;
+
+  it('can be created and disposed', function () {
+    let envelope = new p5.Envelope();
+    envelope.dispose();
+  });
+
+  it('Testing setRange', function () {
+    let envelope = new p5.Envelope();
+    envelope.setRange(5, 6);
+    expect(envelope.aLevel).to.be.equal(5);
+    expect(envelope.rLevel).to.be.equal(6);
+    envelope.dispose();
+  });
+
+  it('Testing _setRampAD and setRampPercentages ', function () {
+    let envelope = new p5.Envelope();
+    envelope._setRampAD(5, 6);
```

Comment:
I'm not sure if we should be testing this method, since it starts with an `_` that convention is the closest thing we have in JavaScript to something like a "private" keyword from Java. It means that the function and/or property shouldn't be called directly outside of the class, it should only be used internally.

We should focus on testing the public methods (which are ideally all documented here https://p5js.org/reference/#/p5.Envelope - I noticed we're missing documentation for `setRampPercentages`)

Reply:
removed